spec/features/review/user_edits_review_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "remove a review for a restaurant", %q(
+  As a user I want to edit a review I added for a given restaurant
+) do
+
+  scenario "Authenticated user unsuccessfully edits a review they made" do
+    restaurant = FactoryGirl.create(:restaurant)
+    user = FactoryGirl.create(:user)
+    sign_in_as(user)
+
+    review_owned_by_user = FactoryGirl.create(
+      :review, restaurant: restaurant, user: user
+    )
+
+    # user needs to click on edit next to the review that belongs to them
+    visit restaurant_review_path(
+      restaurant, review_owned_by_user, method: :patch
+    )
+
```

Comment:
Same comment as above. Refactor all these tests with the suggestions I made and I will take a look at them.